app/page.tsx

```diff
@@ -70,14 +70,28 @@ export default function Home() {
               </p>
             </div>
 
-            <div className="bg-white p-6 rounded-lg shadow-sm transition-all duration-300 hover:bg-orange-50 hover:shadow-md cursor-pointer">
+            <a
+              href="https://certify.sliitmozilla.org"
+              target="_blank"
+              rel="noopener noreferrer"
+              className="bg-white p-6 rounded-lg shadow-sm transition-all duration-300 hover:bg-orange-50 hover:shadow-md cursor-pointer block"
+            >
               <span className="text-orange-600 text-2xl block text-left mb-2">⁕</span>
-              <h3 className="text-orange-600 font-bold mb-2">Community</h3>
+              <h3 className="text-orange-600 font-bold mb-2">Awards and Prizes</h3>
               <p className="text-gray-600">
-                Engage with a supportive network of peers, mentors, and tech enthusiasts eager to
-                share knowledge.
+                Earn recognition through certificates and awards for your contributions and
+                achievements.
+                <br />
+                <a
```

Comment:
React will warn about a hydration issue if you nest an `a` inside of another `a`.
https://nextjs.org/docs/messages/react-hydration-error

We can remove the hyperlink set to the card. So only the text will work as a link.

Additionally, I'd like a bit of space before the link. It feels to tight there right now. Might also add an icon to make it more UX friendly. Go with the [`external-link` icon](https://lucide.dev/icons/external-link) from lucide.